src/Compilers/CSharp/Test/Syntax/Syntax/SeparatedSyntaxListTests.cs

```diff
@@ -44,11 +44,15 @@ public void EnumeratorEquality()
             Assert.Throws<NotSupportedException>(() => default(SeparatedSyntaxList<CSharpSyntaxNode>.Enumerator).Equals(default(SeparatedSyntaxList<CSharpSyntaxNode>.Enumerator)));
         }
 
-        [WorkItem(308077, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/308077")]
-        [Fact]
+        [Fact, WorkItem("http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/308077")]
         public void TestSeparatedListInsert()
         {
-            var list = SyntaxFactory.SeparatedList<ExpressionSyntax>();
+            TestSeparatedListInsertWorker(SyntaxFactory.SeparatedList<ExpressionSyntax>());
+            TestSeparatedListInsertWorker([]);
```

Comment:
to add tests, i went to all the SyntaxFactory methods that create the different list types and i went and found where they were used in tests.  i then tried to update at least two tests.  A test that called the empty factory method, and tests that called the singleton or multi item factory methods.